IND/MarketIndices/BasketIndices.rdf

```diff
@@ -135,7 +136,7 @@
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">credit index</rdfs:label>
 		<skos:definition xml:lang="en">reference index that is a function of credit events that change the value of an underlying portfolio</skos:definition>
```

Comment:
Definition of CapitalizationBasedWeightingFunction should replace " movement in" by "relative".
And "time" should be "times" or "multiplied by"
And "the index" should be "an index"

Reply:
Done